app/views/orders/index.html.erb

```diff
@@ -4,10 +4,12 @@
   <table>
     <tr>
       <th>Order Number</th>
+      <th></th>
```

Comment:
I think that makes sense. I was playing with it on local and noticed I set up the table a little wonky. The table and th tags should probably come before the block. I was looking at some materialize stuff and thought this table might look good with the 'striped' class.